packages/cos/build.yaml

```diff
@@ -32,6 +23,27 @@ requires:
   category: "system"
   version: ">=0"
 
+# https://luet-lab.github.io/docs/docs/concepts/packages/specfile/#copy
+copy:
+- package: 
+    category: "toolchain"
+    name: "yip"
+    version: ">=0"
+  source: "/usr/bin/yip"
+  destination: "/usr/bin/yip"
```

Comment:
Out of curiosity,  are `source` and `destination` mandatory? I would expect that without them the `copy` procedure just untars the package contents in `/`. Is that the case? And if so, why aren't we using it here? `yip` package only includes `/usr/bin/yip` after all.

Reply:
`copy` is meant to be a low-level function, it indeed just transliterates to `COPY ...` internally, but it makes sure there is a build image for the underlying package. 

For package contents we have a separate issue https://github.com/mudler/luet/issues/173, but the problem there is about mix and match: that is to say, at that point, or you use `requires` or the final images

Reply:
ok - turns out that since `copy` is there already it's really straightforward.. so I've tried it and works, I'll push it forward in the next release

Reply:
`0.16.0` should include support for `join`, https://github.com/mudler/luet/releases/tag/0.16.0